src/extension/app-schema/app-schema-test/src/test/resources/test-data/stations/multiValues/stations.xml

```diff
@@ -84,9 +84,9 @@
         <AttributeMapping>
           <targetAttribute>st_${GML_PREFIX}:info</targetAttribute>
           <jdbcMultipleValue>
-            <sourceColumn>ID</sourceColumn>
+            <sourceColumn>info_id</sourceColumn>
```

Comment:
Probably this already existing scenario is used on another former test.  Since this is a bug fix we should add a new scenario without modify or override existing ones.

Reply:
fixed